charts/sn-operator/Chart.yaml

```diff
@@ -25,7 +25,7 @@ appVersion: "v0.4.0"
 
 # This is a semver range of compatible Kubernetes versions. Helm will validate the version
 # constraints when installing the chart and fail if the cluster runs an unsupported Kubernetes version
-kubeVersion: ">= 1.16.0-0"
+kubeVersion: ">= 1.27.0-0"
```

Comment:
Should skip this change since there is another PR updated to 1.17. And 1.27 I think it's a strict limitation for users.